### ISSUE TITLE
Gem litters the default namespace

### ISSUE THREAD

**Issue by bryanrite** [donnen/TaxCloud#1]
The way the gem is structured, it uses the default gem namespace with very generic names.  This can very easily cause conflicts and makes `requires` ambiguous.

For example, to use, I have to add these requires to the top of the class:

``` ruby
require 'address'
require 'cartitem'
require 'service/lookup'
require 'response/response_base'
```

A recommended best practice is to include a single include of the gem's name which then includes everything that is subclassed:

``` ruby
require 'tax_cloud'
```

and `tax_cloud` will require all the appropriate files from there.

Take a look at the `savon` gem (or pretty much any other gem) and see how they've structured their `lib/` directory.  It makes for much cleaner code.

I would do a pull request, but this is a pretty major change to the way this gem works, is this something you guys are willing to implement/use?

**Comment by donnen** [comment]
Hi, thanks for the feedback. We will definitely include these changes in the next version which is being worked on now.

**Comment by donnen** [comment]
Resolved in v1.0.1. Thanks.
